lib/javascripts/base_app.js

```diff
@@ -55,17 +48,26 @@ function BaseApp(zafClient, data) {
   this.I18n = { t: I18n.t };
   registerHelpers(this);
   bindEvents(this);
-  let evt = { firstLoad: true };
+
   this._metadata = data.metadata;
   this._context = data.context;
-  zafClient.get('currentUser.locale').then(function(data) {
-    I18n.loadTranslations(data['currentUser.locale']);
```

Comment:
Moved to the initializer as it may be needed when writing apps from scratch: https://github.com/zendesk/app_scaffold/pull/16/files#diff-1264142ccf3376b73bec45e04675f8fcR8